ts/src/index.ts

```diff
@@ -0,0 +1,74 @@
+import * as anchor from "@project-serum/anchor";
+import { BN } from "@project-serum/anchor";
+import { PublicKey } from "@solana/web3.js";
+
+export const CHAINLINK_AGGREGATOR_PROGRAM_ID = new PublicKey(
+  "HW3ipKzeeduJq6f1NqRCw4doknMeWkfrM4WxobtG3o5v"
+);
+export const CHAINLINK_STORE_PROGRAM_ID = new PublicKey(
+  "CaH12fwNTKJAG8PxEvo9R96Zc2j8qNHZaFj8ZW49yZNT"
+);
```

Comment:
doesn't this need to be updated to the correct IDs?

Reply:
Good catch, updated